src/notify.js

```diff
@@ -59,6 +63,14 @@ function Notify(title, options = {}) {
     }
 }
 
+Notify.serviceWorkerSupported = function() {
```

Comment:
I think we could add a convenience method that feature detects support for it at least. Seems like most users will want to use it as a dependency for doing notifications via SW (if push is not supported, how useful would notifications be anyway?)